pages/api/v1/acmurl/generate.ts

```diff
@@ -1,11 +1,52 @@
-import type { Data } from "@/util/types";
-import type { NextApiRequest, NextApiResponse } from "next";
+import type {Data} from "@/util/types";
+import type {NextApiRequest, NextApiResponse} from "next";
+import NextCors from "nextjs-cors";
+import {addACMURL, handleExistingACMURL} from "@/util/acmurl";
+import {verifyAuth} from "@/util";
 
 /**
- * TODO
- * @param req
- * @param res
+ * ACM URL generation endpoint. This endpoint is used to generate ACM URLs. Parameters for request:
```

Comment:
Another nitpick: 'ACMURL' instead of 'ACM URL' here and in the rest of this file I think. Could we also explain what ACMURL is here so new readers aren't super confused? (link shortener using YOURLS, etc.)